### ISSUE TITLE
Can't install Gemini in mac

### ISSUE THREAD

**Issue by DenisBarabanshchikov** [gemini-testing/gemini#972]
node v12.6.0
After npm install -g gemini

`/usr/local/bin/gemini -> /usr/local/lib/node_modules/gemini/bin/gemini

> png-img@2.2.0 install /usr/local/lib/node_modules/gemini/node_modules/png-img
> node-gyp rebuild

  CC(target) Release/obj.target/libpng/third_party/libpng/png.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngerror.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngget.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngmem.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngpread.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngread.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrio.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrtran.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrutil.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngset.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngtest.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngtrans.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwio.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwrite.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwtran.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwutil.o
  LIBTOOL-STATIC Release/png.a
  CC(target) Release/obj.target/zlib/third_party/zlib/adler32.o
  CC(target) Release/obj.target/zlib/third_party/zlib/compress.o
  CC(target) Release/obj.target/zlib/third_party/zlib/crc32.o
  CC(target) Release/obj.target/zlib/third_party/zlib/deflate.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzclose.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzlib.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzread.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzwrite.o
  CC(target) Release/obj.target/zlib/third_party/zlib/infback.o
  CC(target) Release/obj.target/zlib/third_party/zlib/inffast.o
  CC(target) Release/obj.target/zlib/third_party/zlib/inflate.o
../third_party/zlib/inflate.c:1507:61: warning: shifting a negative signed value
      is undefined [-Wshift-negative-value]
    if (strm == Z_NULL || strm->state == Z_NULL) return -1L << 16;
                                                        ~~~ ^
1 warning generated.
  CC(target) Release/obj.target/zlib/third_party/zlib/inftrees.o
  CC(target) Release/obj.target/zlib/third_party/zlib/trees.o
  CC(target) Release/obj.target/zlib/third_party/zlib/uncompr.o
  CC(target) Release/obj.target/zlib/third_party/zlib/zutil.o
  LIBTOOL-STATIC Release/zlib.a
  CXX(target) Release/obj.target/png_img/src/init.o
../src/init.cc:10:125: error: too few arguments to function call, single
      argument 'context' was not specified
  ...New<FunctionTemplate>(PngImgAdapter::NewInstance)->GetFunction());
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/User/.node-gyp/12.6.0/include/node/v8.h:5948:3: note: 
      'GetFunction' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<Function> GetFunction(
  ^
/User/.node-gyp/12.6.0/include/node/v8config.h:351:31: note: 
      expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
1 error generated.
make: *** [Release/obj.target/png_img/src/init.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 18.6.0
gyp ERR! command "/usr/local/Cellar/node/12.6.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/gemini/node_modules/png-img
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! png-img@2.2.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the png-img@2.2.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /User/.npm/_logs/2019-07-28T08_37_17_939Z-debug.log`

**Comment by j0tunn** [comment]
`png-img` doesn't compile for node@12: https://github.com/gemini-testing/png-img/issues/45
Use node@8-10 for now

**Comment by j0tunn** [comment]
Fixed in `png-img@2.2.1`
